deploy

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+echo $HOSTNAME  # should be installing on calim
+echo $PWD  # should be base of code repo
+whoami  # should be user pipeline (or ubuntu?)
+
+# setup
+DATE=$(date --iso-8601=seconds)
+LOG=~/deploy_lwa-antpos_$DATE.log
+
+if [[ $HOSTNAME =~ 'calim' ]]; then
+    echo "$DATE" >> $LOG 2>>$LOG
```

Comment:
the group needs to discuss how growing log files should be managed. If it's the script's responsibility then the script needs logic to either rotate log files or truncate them.